### ISSUE TITLE
[ASPagerNode] Cell node layout not resizing after rotation

### ISSUE THREAD

**Issue by mohitnandwani** [TextureGroup/Texture#1753]
New to texture and using Texture version 2.8.1. I tried adding Page layout to my project. The portrait layout works completely fine but whenever I rotate my device cell's layout don't resize according to node's bounds. I'm attaching screenshots and a sample project. By now, I've tried adjusting the scroll inset and transitioning layout on `viewWillTransition` but had no luck. Let me know if a solution for this exist.

**Portrait Mode:**
![Portrait](https://user-images.githubusercontent.com/28500428/71545365-4c576400-29b0-11ea-8999-523ebdae1b95.png)

**Landscape Mode:**
![Landscape](https://user-images.githubusercontent.com/28500428/71545371-55483580-29b0-11ea-8efc-fb34e9fb1497.png)

**Example Project:**
[PagerNodeRoationIssue.zip](https://github.com/TextureGroup/Texture/files/4007149/PagerNodeRoationIssue.zip)

**Comment by foxware00** [comment]
I'm facing similar but subtly different issues with measuring pager node during rotation.

Is there any known issues/workarounds @nguyenhuy?

**Comment by foxware00** [comment]
@mohitnandwani Did you get anywhere with layout issues?
I'm getting closer, but safeAreaInsets are being incorrectly reported the first time I draw a lot of views in landscape

**Comment by mohitnandwani** [comment]
> @mohitnandwani Did you get anywhere with layout issues?
> I'm getting closer, but safeAreaInsets are being incorrectly reported the first time I draw a lot of views in landscape

Nope. Still having issues with resizing. How's your progress? Can you share an example code base of yours?

**Comment by mohitnandwani** [comment]
@foxware00 One observation I've had is that if i reload the cells, then only layout updates.

**Comment by foxware00** [comment]
@mohitnandwani I've settled on consuming the safe areas before laying out the pager and insetting the whole view and then making the view controllers (each page of the pager) ignore safe areas as this is where Texture has a bug. I haven't been able to work out why yet. As for resizing I just call relayoutItems when `viewWillTransition` is called on the parent view controller. Thus they they seem to size correctly. It's just the safe areas that have the issue
